toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -44,6 +44,8 @@ curl-7.76.0-5.cm2.aarch64.rpm
 curl-debuginfo-7.76.0-5.cm2.aarch64.rpm
 curl-devel-7.76.0-5.cm2.aarch64.rpm
 curl-libs-7.76.0-5.cm2.aarch64.rpm
+debugedit-5.0-1.cm2.aarch64.rpm
```

Comment:
Does debugedit need to be in the worker chroot by default? If so the update_manifests.sh script should be updated as well

Reply:
Looking further below, it looks like rpm now requires debugedit. So adding here makes sense. But please also update that script.